index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Simple Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body class="start">
+    <main>
+      <header class="weather_header">
+        <h1>Weather Report</h1>
+        <h5>
+          For the lovely city of
+          <span class="header_city"></span>
```

Comment:
You could put a default city name here so that it won't be blank when the page loads.